h2o-algos/src/test/java/hex/tree/isoforextended/ExtendedIsolationForestTest.java

```diff
@@ -0,0 +1,264 @@
+package hex.tree.isoforextended;
+
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.fvec.TestFrameBuilder;
+import water.fvec.Vec;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ExtendedIsolationForestTest extends TestUtil {
+
+    @BeforeClass()
+    public static void setup() {
+        stall_till_cloudsize(1);
+    }
+
+    @Test
+    public void testBasic() {
+        try {
+            Scope.enter();
+            Frame train = Scope.track(parse_test_file("smalldata/anomaly/single_blob.csv"));
+
+            ExtendedIsolationForestModel.ExtendedIsolationForestParameters p =
+                    new ExtendedIsolationForestModel.ExtendedIsolationForestParameters();
+            p._train = train._key;
+            p._seed = 0xDECAF;
+            p._ntrees = 100;
+            p.extension_level = train.numCols() - 1;
+
+            ExtendedIsolationForest eif = new ExtendedIsolationForest(p);
+            ExtendedIsolationForestModel model = eif.trainModel().get();
+            Scope.track_generic(model);
+            assertNotNull(model);
+
+            Frame test = Scope.track(parse_test_file("smalldata/anomaly/single_blob.csv"));
+            Frame out = model.score(test);
```

Comment:
Do you expect some specific output to test it? For example, in comparison to the implementation of the paper?

Reply:
No, I can test the output values itself but it depends on seed and branching implementation.